### PR TITLE
navbar-Item

### PR DESCRIPTION
**Pull request by JaqKent**
## Summary

Add Navigation Bar links

## Screenshots

image

## Trello Card

https://trello.com/c/l57tMfOD

### CODE REVIEW

File: src/components/NavigationBar/components/NavbarLink/components/i18n.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  whereToinvest: 'Donde invertir',
+  aboutUs: 'Quienes somos',
+  sellAProperty: 'Vender una propiedad',
+  frequentQuestions: 'Preguntas recuentes'
+});
+
+i18next.addResources('en', 'NavItems', {
+  principal: 'Principal',
+  whereToinvest: 'Where to invest',
+  aboutUs: 'Aboust us',
```
Comment (nathan-hyan) [inline src/components/NavigationBar/components/NavbarLink/components/i18n.ts]: Typo here

File: src/components/NavigationBar/components/NavbarLink/components/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    cursor: pointer;
+  }
+  &:active{
+    color: darken($primary, 10);
+  }
+}
+
+.active {
+  color: darken($primary, 10);
+}
```
Comment (nathan-hyan) [inline src/components/NavigationBar/components/NavbarLink/components/styles.module.scss]: Both lighten and dark variations as they are being repeated, may be better if they're a new variable

File: src/components/NavigationBar/components/NavbarLink/styles.module.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+@import "scss/variables/_colors";
+@import "scss/variables/_mixins";
+
+.downside{
+@include flex(center,center,row);
+padding: 15px;
+
+  border-bottom: 1px solid $black;
+ 
+}
```
Comment (nathan-hyan) [inline src/components/NavigationBar/components/NavbarLink/styles.module.scss]: Prettier this file

File: src/screens/Home/index.tsx [hunk 4]
```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 
+import NavBarItems from 'components/NavigationBar/components/NavbarLink';
+
```
Comment (nathan-hyan) [inline src/screens/Home/index.tsx]: Remove empty space

**Review by nathan-hyan — CHANGES_REQUESTED** [review]

File: src/screens/Home/index.tsx (hunk 4 above)
Comment (JaqKent) [reply src/screens/Home/index.tsx]: done

File: src/components/NavigationBar/components/NavbarLink/styles.module.scss (hunk 3 above)
Comment (JaqKent) [reply src/components/NavigationBar/components/NavbarLink/styles.module.scss]: done

File: src/components/NavigationBar/components/NavbarLink/components/styles.module.scss (hunk 2 above)
Comment (JaqKent) [reply src/components/NavigationBar/components/NavbarLink/components/styles.module.scss]: done

**Review by nathan-hyan — APPROVED** [review]
🥇

**Pull request merged**